cdap-client-tests/pom.xml

```diff
@@ -63,6 +63,12 @@ the License.
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-data-fabric</artifactId>
```

Comment:
This is required so that copied Tephra classes take precedence over tephra jars. This is required to make the unit tests pass.